docs/source/conf.py

```diff
@@ -16,12 +14,7 @@
 import time
 import contextlib
 
-from aiida.manage.configuration import load_documentation_profile
-
 import aiida_testing
-
-load_documentation_profile()
```

Comment:
This was removed since it is no longer needed in AiiDA 2.0

Reply:
This was dropped at 2.5.0: https://github.com/aiidateam/aiida-core/blob/main/CHANGELOG.md#v250---2023-12-20